src/avfoundation.cs

```diff
@@ -10869,8 +10869,9 @@ interface AVPlayerItem : NSCopying {
 		[Export ("navigationMarkerGroups", ArgumentSemantic.Copy)]
 		AVNavigationMarkersGroup[] NavigationMarkerGroups { get; set; }
 
-		[NoiOS][NoMac]
+		[NoMac]
 		[TV (9,0)]
+		[iOS (12,0)]
```

Comment:
This is new in iOS 12? Not 13? Ideally you'd need to run the introspection tests on an iOS 12 simulator to confirm (you can do so by opening the `test.sln`, select introspection as a target (top left) and deploy on an iOS 12 sim (you might need to go to Xcode's preferences and download an older simulator under `Components`).

Reply:
Yes I found it strange also but : https://developer.apple.com/documentation/avfoundation/avplayeritem/1627623-externalmetadata

Reply:
I ran the introspection test and an error occurred, I change it to ``` iOS (13,0)```:

![](https://files.gitter.im/xamarin/xamarin-macios/S8bR/1F7DB7FF-0B8E-48D5-98EA-FE0C28FB64FC.png)